pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -97,9 +103,30 @@
   social_maker_title: "I'm a Maker!"
   social_maker_desc: "Code.org physical computing and maker resources for grades 6-12."
 
+  social_blockchain_title: "How Blockchain Works"
+  social_blockchain_desc: "Videos and lessons that explore what blockchain is and why it's important."
+
   pl_superhero_title: "Help students become superheroes!"
   pl_superhero_desc: "Join our highly supportive Professional Learning Program for middle and high school educators."
 
+  blockchain_hero_heading: 'How Blockchain Works'
+  blockchain_hero_desc: 'Dive into the world of blockchain with "How Blockchain Works," our new video series and accompanying lessons!'
+  blockchain_intro_heading: 'See how blockchain works'
+  blockchain_intro_desc: 'The video series, made in partnership with Coinbase, features industry experts and aims to demystify this technology.'
+  blockchain_intro_list_heading: 'This series will:'
```

Comment:
I think the eng team needs to work with the international team to norm on what to do in instances like this –– "This series will:" may not be translatable without knowing more context (What is a series in this context? What form does the verb take without knowing the object (which doesn't matter in English)?).

I'm going to be starting on something soon that will raise similar questions. I don't think there's any action you need to take right now, but flagging it for future conversation.

Reply:
Tagging @castro-jorge for eyes on this. Nothing to do right now, I think